pkg/server/server.go

```diff
@@ -175,7 +175,7 @@ func WithTransport(t gateway.Transport) OpenFGAServiceV1Option {
 	}
 }
 
-// WithResolveNodeLimit sets a limit on the number of recursive calls that one Check or ListObjects call will allow.
+// WithResolveNodeLimit sets a limit on the number of recursive calls that one Check, ListObjects or ListUsers call will allow.
```

Comment:
nit (non-blocking): Make the godoc more generic so we can avoid the specifics that may change/grow over time.

> ... sets a limit on the number of recursive calls that one FGA query will allow. ...

Reply:
Not a bad suggestion but IMO the specificity of the comment is important here. Also, don't think we'll be adding more applicable endpoints for a while.